3506/3/app/views/index.erb

```diff
@@ -0,0 +1,14 @@
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
ERB is a preprocessor, so it cannot be a type of file. This file should be named in the next manner `index.html.erb`